students/Jaidjen/lesson2/Lesson2FizzBuzz.py

```diff
@@ -0,0 +1,9 @@
+for number in range(1,101):
+    if number % 3 is 0:
```

Comment:
use `is` only with objects, for example `is True` `is None`

for primitive values comparison use `==`